app/controllers/api/v1/monsters_controller.rb

```diff
@@ -19,9 +21,26 @@ def create
     end
   end
 
-  private
+  def destroy
+    Monster.find(params[:id]).destroy
+    render json: {}, status: :no_content
+  end
+
+  protected 
 
   def monster_params
     params.require(:monster).permit(:name, :description, :classification, :habitat)
   end
+
+  def authenticate_user
+    if !user_signed_in?
+      render json: {error: ["You need to be signed in first!"]}
+    end
+  end
+
+  def authorize_user
+    if !user_signed_in? || !current_user.admin?
+      render json: {error: ["You are not authorized to do that!"]}
+    end
+  end
```

Comment:
These user methods would be great within the `ApplicationController` so that other controllers can use them as well!